client/datainfo/model.go

```diff
@@ -0,0 +1,20 @@
+package datainfo
+
+type User struct {
+	Username string `json:"username"`
+	Zone     string `json:"zone"`
+}
+
+type Group struct {
+	Name    string   `json:"name,omitempty"`
+	Members []string `json:"members"`
+}
+
+type ServiceError struct {
+	ErrorCode string `json:"error_code"`
+
+	Reason any
+	User   string
+	Users  []string
+	Group  string
```

Comment:
These are in here solely so that the debug logging will show them if it's turned on. The error code is the only thing used by non-debugging code.